aten/src/ATen/hip/impl/HIPCachingAllocatorMasqueradingAsCUDA.h

```diff
@@ -4,11 +4,15 @@
 #include <ATen/hip/impl/HIPAllocatorMasqueradingAsCUDA.h>
 #include <ATen/hip/impl/HIPStreamMasqueradingAsCUDA.h>
 
-namespace c10 { namespace hip {
+namespace c10 {
+// forward declaration
+class DataPtr;
```

Comment:
Question: why this in in c10 namespace instead of HIPCachingAllocatorMasqueradingAsCUDA namespace?